src/main/java/com/max/grpc/orders/server/controllers/OrdersController.java

```diff
@@ -0,0 +1,29 @@
+
+package com.max.grpc.orders.server.controllers;
+
+import com.max.grpc.orders.proto.Order;
+import com.max.grpc.orders.proto.OrderReceipt;
+import com.max.grpc.orders.proto.OrderServiceGrpc;
+import com.max.grpc.orders.server.services.OrdersService;
+
+import io.grpc.stub.StreamObserver;
+import org.apache.log4j.Logger;
+
+public class OrdersController extends OrderServiceGrpc.OrderServiceImplBase {

```

Comment:
rename to something different not controller

Reply:
Renamed back to `***GrpcService`